spring-cloud-alibaba-sentinel/src/main/java/org/springframework/cloud/alibaba/sentinel/custom/SentinelBeanPostProcessor.java

```diff
@@ -60,10 +76,69 @@ public void postProcessMergedBeanDefinition(RootBeanDefinition beanDefinition,
 				sentinelRestTemplate = beanDefinition.getResolvedFactoryMethod()
 						.getAnnotation(SentinelRestTemplate.class);
 			}
+			// check class and method validation
+			checkSentinelRestTemplate(sentinelRestTemplate, beanName);
 			cache.put(beanName, sentinelRestTemplate);
 		}
 	}
 
+	private void checkSentinelRestTemplate(SentinelRestTemplate sentinelRestTemplate,
+			String beanName) {
+		checkBlock4RestTemplate(sentinelRestTemplate.blockHandlerClass(),
+				sentinelRestTemplate.blockHandler(), beanName,
+				SentinelConstants.BLOCK_TYPE);
+		checkBlock4RestTemplate(sentinelRestTemplate.fallbackClass(),
+				sentinelRestTemplate.fallback(), beanName,
+				SentinelConstants.FALLBACK_TYPE);
+	}
+
+	private void checkBlock4RestTemplate(Class<?> blockClass, String blockMethod,
+			String beanName, String type) {
+		if (blockClass == void.class && StringUtils.isEmpty(blockMethod)) {
+			return;
+		}
+		if (blockClass != void.class && StringUtils.isEmpty(blockMethod)) {
+			logger.error(
+					"{} class property exists but {}"
+							+ " method property is not exists in bean[{}]",
+					type, type, beanName);
+			System.exit(-1);
+		}
+		else if (blockClass == void.class && !StringUtils.isEmpty(blockMethod)) {
+			logger.error(
+					"{} method property exists but {} class property is not exists in bean[{}]",
+					type, type, beanName);
+			System.exit(-1);
```

Comment:
Too violent to use System.exit(-1)，maybe just throw Exception is better

Reply:
yes, fixed now.